packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -71,6 +71,10 @@ import ParameterError from '../common/errors/parameter';
 import MediaError from '../common/errors/media';
 import {MeetingInfoV2PasswordError, MeetingInfoV2CaptchaError} from '../meeting-info/meeting-info-v2';
 import BrowserDetection from '../common/browser-detection';
+import {ReceiveSlotManager} from '../multistream/receiveSlotManager';
```

Comment:
majority of the changes in this file are related to creating RemoteMediaManager and forwarding the events from it - this is not confirmed if it will stay like this, so for now I've raised a jira to add the unit tests once this is settled: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-385754